src/plugins/intel_gna/src/transformations/insert_identity_layer.cpp

```diff
@@ -231,3 +231,29 @@ bool IdentityCandidatesCleanup::run_on_model(const std::shared_ptr<ov::Model>& f
     }
     return false;
 }
+
+bool InsertIdentityForNonQuantizableConcatInput::insert_identity_for_non_quantizable_concat_inputs(
+    const std::shared_ptr<ov::Node>& node) {
+    bool is_graph_modified = false;
+    for (size_t i = 0; i < node->get_input_size(); i++) {
+        auto concat_input_node = node->get_input_node_shared_ptr(i);
+        auto prev_node = get_prev_node_skipping_certain(concat_input_node, is_gna_not_quantizable);
+        if (is_parameter(concat_input_node) || is_parameter(prev_node)) {
+            insert_identity_layer_after(concat_input_node, 0);
```

Comment:
is 0 proper value?

Reply:
Good question. Added method for calculation of proper value.